src/base/datetime/calendar/header/index.ts

```diff
@@ -0,0 +1,295 @@
+import { html, property, svg, query } from "lit-element";
+import {
+  KucBase,
+  dispatchCustomEvent,
+  CustomEventDetail
+} from "../../../kuc-base";
+import { en, zh, ja } from "../../resource/locale";
+
+type BaseDateTimeCalendarHeaderProps = {
+  language?: "en" | "zh" | "ja";
+  month?: number;
+  year?: number;
+};
+
+export class BaseDateTimeCalendarHeader extends KucBase {
+  @property({ type: String }) language = "en";
+  @property({ type: Number }) month = 1;
+  @property({ type: Number }) year = 2021;
+
+  @query(".kuc-base-datetime-calendar-header__group__year")
+  private _yearSelectEl: HTMLSelectElement | undefined;
+
+  @query(".kuc-base-datetime-calendar-header__group__month")
+  private _monthSelectEl: HTMLSelectElement | undefined;
+
+  constructor(props?: BaseDateTimeCalendarHeaderProps) {
+    super();
+    if (!props) {
+      return;
+    }
+    this.language =
+      props.language !== undefined ? props.language : this.language;
+    this.month = props.month !== undefined ? props.month : this.month;
+    this.year = props.year !== undefined ? props.year : this.year;
+  }
+
+  private _handleClickCalendarHeaderButtonPreviousMonth(event: MouseEvent) {
```

Comment:
@dong0 
There is only one previous month button, i think we can change the shorter name:
_handleClickCalendarPrevMonthBtn